src/rules.ts

```diff
@@ -115,19 +115,39 @@ export const rules: Rule[] = [
       'Trailing spaces',
       'Removes extra spaces after every line.',
       RuleType.SPACING,
-      (text: string) => {
-        return text.replace(/[ \t]+$/gm, '');
+      (text: string, options = {}) => {
+        if (options['TwoSpaceLinebreak'] === false) {
+          return text.replace(/[ \t]+$/gm, '');
+        } else {
+          text = text.replace(/\b[ \t]$/gm, ''); // one whitespace
+          text = text.replace(/\b[ \t]{3,}$/gm, ''); // three or more whitespaces
+          text = text.replace(/\b( ?\t ?)$/gm, ''); // two whitespaces with at least one tab
+          return text;
+        }
       },
       [
         new Example(
             'Removes trailing spaces and tabs',
             dedent`
         # H1   
-        line with trailing spaces and tabs            `, // eslint-disable-line no-tabs
+        line with trailing spaces and tabs	        `, // eslint-disable-line no-tabs
             dedent`
         # H1
         line with trailing spaces and tabs`,
         ),
+        new Example(
+            'With `TwoSpaceLinebreak=true`',
+            dedent`
+        # H1
+        line with trailing spaces and tabs  `,
+            dedent`
+        # H1
+        line with trailing spaces and tabs  `,
+            {TwoSpaceLinebreak: true},
+        ),
+      ],
+      [
+        new BooleanOption('TwoSpaceLinebreak', 'Ignore two spaces followed by a line break ("Two Space Rule").', false),
```

Comment:
can rename to `Two Space Linebreak` for readability

Reply:
will do